### PR TITLE
Update to support Grant to Group syntax

### PR DESCRIPTION
**Pull request by JensTrier**
Updating some Grant PS cmdlet pages to show grant to group syntax.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: skype/skype-ps/skype/Grant-CsCallingLineIdentity.md [hunk 1]
```diff
@@ -13,21 +13,27 @@ ms.reviewer:
 # Grant-CsCallingLineIdentity
 
 ## SYNOPSIS
-Use the `Grant-CsCallingLineIdentity` cmdlet to apply a Caller ID policy to a user account or set the tenant Global instance.
+Use the `Grant-CsCallingLineIdentity` cmdlet to apply a Caller ID policy to a user account, to a group of users or set the tenant Global instance.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Grant-CsCallingLineIdentity.md:16]:
```suggestion
Use the `Grant-CsCallingLineIdentity` cmdlet to apply a Caller ID policy to a user account, to a group of users, or to set the tenant Global instance.
```

File: skype/skype-ps/skype/Grant-CsCallingLineIdentity.md [hunk 2]
```diff
@@ -13,21 +13,27 @@ ms.reviewer:
...
+```
+
 ### Identity
 ```
 Grant-CsCallingLineIdentity [[-Identity] <string>] [[-PolicyName] <string>] [-PassThru] [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-You can either assign a Caller ID policy to a specific user or you can set the Global policy instance.
+You can either assign a Caller ID policy to a specific user, to a group of users or you can set the Global policy instance.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Grant-CsCallingLineIdentity.md:36]:
```suggestion
You can either assign a Caller ID policy to a specific user, to a group of users, or you can set the Global policy instance.
```

File: skype/skype-ps/skype/Grant-CsDialoutPolicy.md [hunk 3]
```diff
@@ -5,19 +5,31 @@ applicable: Skype for Business Online, Microsoft Teams
...
+author: jenstrier
+ms.author: jenstr
 ms.reviewer:
 ---
 
 # Grant-CsDialoutPolicy
 
 ## SYNOPSIS
-Use the `Grant-CsDialoutPolicy` cmdlet to assign the tenant global or a per-user outbound calling restriction policy to one or more users.
+Use the `Grant-CsDialoutPolicy` cmdlet to assign the tenant global, a group of users or a per-user outbound calling restriction policy to one or more users.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Grant-CsDialoutPolicy.md:16]:
```suggestion
Use the `Grant-CsDialoutPolicy` cmdlet to assign the tenant global, a group of users, or a per-user outbound calling restriction policy to one or more users.
```

**Review by dariomws — APPROVED** [review]

File: skype/skype-ps/skype/Grant-CsDialoutPolicy.md [hunk 4]
```diff
@@ -26,43 +38,51 @@ To get all the available policies in your organization run `Get-CsOnlineDialOutP
...
+
+This example assigns the policy instance "InternationalAndDomestic" to the members of the group support@contoso.com.
+
+
 ## PARAMETERS
 
-### -Confirm
-Prompts you for confirmation before running the cmdlet.
+### -Global
+This parameter set the tenant global policy instance. This is the policy that all users in the tenant will get, unless they have a specific policy instance assigned.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Grant-CsDialoutPolicy.md:80]:
```suggestion
This parameter sets the tenant global policy instance. This is the policy that all users in the tenant will get unless they have a specific policy instance assigned.
```

File: skype/skype-ps/skype/Grant-CsOnlineVoiceRoutingPolicy.md [hunk 5]
```diff
@@ -5,28 +5,31 @@ applicable: Microsoft Teams, Skype for Business Online
...
+author: jenstrier
+ms.author: jenstr
 ms.reviewer:
 ---
 
 # Grant-CsOnlineVoiceRoutingPolicy
 
 ## SYNOPSIS
-Assigns a per-user online voice routing policy to one or more users. Online voice routing policies manage online PSTN usages for Phone System users.
+Assigns a per-user online voice routing policy to one user, a group of users or sets the Global policy instance. Online voice routing policies manage online PSTN usages for Phone System users.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Grant-CsOnlineVoiceRoutingPolicy.md:16]:
```suggestion
Assigns a per-user online voice routing policy to one user, a group of users, or sets the Global policy instance. Online voice routing policies manage online PSTN usages for Phone System users.
```

File: skype/skype-ps/skype/Grant-CsOnlineVoiceRoutingPolicy.md [hunk 6]
```diff
@@ -36,72 +39,64 @@ Note that simply assigning a user an online voice routing policy will not enable
...
-Default value: None
-Accept pipeline input: False
-Accept wildcard characters: False
-```
+## PARAMETERS
 
-### -DomainController
-This parameter is reserved for internal Microsoft use.
+### -Identity
+Indicates the Identity of the user account to be assigned the per-user online voice routing policy. User Identities can be specified using one of the following formats: 1) the user's SIP address; 2) the user's user principal name (UPN); or, 3) the user's Active Directory display name (for example, Ken Myer).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Grant-CsOnlineVoiceRoutingPolicy.md:80]:
```suggestion
Indicates the Identity of the user account to be assigned the per-user online voice routing policy. User Identities can be specified using one of the following formats: the user's SIP address, the user's user principal name (UPN), or the user's Active Directory display name (for example, Ken Myer).
```

File: skype/skype-ps/skype/Grant-CsOnlineVoiceRoutingPolicy.md [hunk 7]
```diff
@@ -111,27 +106,29 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -Identity
-Indicates the Identity of the user account to be assigned the per-user online voice routing policy. User Identities can be specified using one of the following formats: 1) the user's SIP address; 2) the user's user principal name (UPN); or, 3) the user's Active Directory display name (for example, Ken Myer).
+### -PolicyName
+"Name" of the policy to be assigned. The PolicyName is simply the policy Identity minus the policy scope (the "tag:" prefix). For example, a policy with the Identity tag:Redmond has a PolicyName equal to Redmond; likewise, a policy with the Identity tag:RedmondOnlineVoiceRoutingPolicy has a PolicyName equal to RedmondOnlineVoiceRoutingPolicy.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Grant-CsOnlineVoiceRoutingPolicy.md:110]:
```suggestion
Name of the policy to be assigned. The PolicyName is simply the policy Identity minus the policy scope (the "tag:" prefix). For example, a policy with the Identity tag:Redmond has a PolicyName equal to Redmond; likewise, a policy with the Identity tag:RedmondOnlineVoiceRoutingPolicy has a PolicyName equal to RedmondOnlineVoiceRoutingPolicy.
```

File: skype/skype-ps/skype/Grant-CsOnlineVoicemailPolicy.md [hunk 8]
```diff
@@ -13,34 +13,46 @@ ms.reviewer:
 # Grant-CsOnlineVoicemailPolicy
 
 ## SYNOPSIS
-Assigns an online voice mail policy at the per-user scope. Online voicemail policies manage usages for Voicemail service.
+Assigns an online voice mail policy to a user account, to a group of users or set the tenant Global instance. Online voicemail policies manage usages for Voicemail service.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Grant-CsOnlineVoicemailPolicy.md:16]:
```suggestion
Assigns an online voicemail policy to a user account, to a group of users, or set the tenant Global instance. Online voicemail policies manage usages for Voicemail service.
```

File: skype/skype-ps/skype/Grant-CsOnlineVoicemailPolicy.md [hunk 9]
```diff
@@ -13,34 +13,46 @@ ms.reviewer:
...
+```
+
 ### Identity
 ```
 Grant-CsOnlineVoicemailPolicy [[-Identity] <string>] [[-PolicyName] <string>] [-PassThru] [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-This cmdlet assigns an existing user-specific online voicemail policy to a user.
+This cmdlet assigns an existing user-specific online voicemail policy to a user, a group of users or the Global policy instance.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Grant-CsOnlineVoicemailPolicy.md:34]:
```suggestion
This cmdlet assigns an existing user-specific online voicemail policy to a user, a group of users, or the Global policy instance.
```

File: skype/skype-ps/skype/Grant-CsTeamsCallParkPolicy.md [hunk 10]
```diff
@@ -40,49 +43,41 @@ The TeamsCallParkPolicy controls whether or not users are able to leverage the c
...
-Default value: None
-Accept pipeline input: False
-Accept wildcard characters: False
-```
+## PARAMETERS
 
-### -DomainController
-Internal Microsoft use only.
+### -Identity
+The User Id of the user to whom the policy is being assigned.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Grant-CsTeamsCallParkPolicy.md:61]:
```suggestion
The User ID of the user to whom the policy is being assigned.
```

File: skype/skype-ps/skype/Grant-CsTeamsCallParkPolicy.md [hunk 11]
```diff
@@ -92,27 +87,29 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -Identity
-The User Id of the user to whom the policy is being assigned.
+### -PolicyName
+"Name" of the policy to be assigned. The PolicyName is simply the policy Identity minus the policy scope ("tag:"). For example, a policy that has the Identity tag:Redmond has a PolicyName equal to Redmond; a policy with the Identity tag:RedmondConferencingPolicy has a PolicyName equal to RedmondConferencingPolicy.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Grant-CsTeamsCallParkPolicy.md:91]:
```suggestion
Name of the policy to be assigned. The PolicyName is simply the policy Identity minus the policy scope ("tag:"). For example, a policy that has the Identity tag:Redmond has a PolicyName equal to Redmond; a policy with the Identity tag:RedmondConferencingPolicy has a PolicyName equal to RedmondConferencingPolicy.
```

File: skype/skype-ps/skype/Grant-CsTeamsCallParkPolicy.md [hunk 12]
```diff
@@ -92,27 +87,29 @@ Accept pipeline input: False
...
+
+If you set PolicyName to a null value, then the command will unassign any per-user policy assigned to the user.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Grant-CsTeamsCallParkPolicy.md:93]:
```suggestion
If you set PolicyName to a null value, the command will unassign any per-user policy assigned to the user. 
```

File: skype/skype-ps/skype/Grant-CsTeamsCallParkPolicy.md [hunk 13]
```diff
@@ -170,19 +165,40 @@ Accept pipeline input: False
...
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
 ### CommonParameters
 This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable.
 For more information, see about_CommonParameters (https://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Grant-CsTeamsCallParkPolicy.md:185]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters(https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: skype/skype-ps/skype/Grant-CsTeamsCallingPolicy.md [hunk 14]
```diff
@@ -15,7 +15,7 @@ ms.reviewer:
 
 ## SYNOPSIS
 
-Cmdlet to assign a specific Teams Calling Policy to a user.
+Cmdlet to assign a specific Teams Calling Policy to a user, a group of users or sets the Global policy instance.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Grant-CsTeamsCallingPolicy.md:18]:
```suggestion
Assigns a specific Teams Calling Policy to a user, a group of users, or sets the Global policy instance.
```

File: skype/skype-ps/skype/Grant-CsTeamsCallingPolicy.md [hunk 15]
```diff
@@ -29,34 +29,46 @@ Grant-CsTeamsCallingPolicy [[-Identity] <string>] [[-PolicyName] <string>] [-Pas
...
 ```
 
+### GrantToGroup
+```
+Grant-CsTeamsCallingPolicy [-Group] <string> [[-PolicyName] <string>] [-PassThru] [-Rank <int>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
 ## DESCRIPTION
-The Teams Calling Policies designate how users are able to use calling functionality within Microsoft Teams. This cmdlet allows admins to grant user level policies to individual users.
+The Teams Calling Policies designate how users are able to use calling functionality within Microsoft Teams. This cmdlet allows admins to grant user level policies to individual users, to members of a group or to set the Global policy instance.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Grant-CsTeamsCallingPolicy.md:38]:
```suggestion
The Teams Calling Policies designate how users are able to use calling functionality within Microsoft Teams. This cmdlet allows admins to grant user level policies to individual users, to members of a group, or to set the Global policy instance.
```

File: skype/skype-ps/skype/Grant-CsTenantDialPlan.md [hunk 16]
```diff
@@ -5,71 +5,102 @@ applicable: Microsoft Teams, Skype for Business Online
...
+author: jenstrier
+ms.author: jenstr
 ms.reviewer:
 ---
 
 # Grant-CsTenantDialPlan
 
 ## SYNOPSIS
-Use the Grant-CsTenantDialPlan cmdlet to assign an existing tenant dial plan to a user.
+Use the Grant-CsTenantDialPlan cmdlet to assign an existing tenant dial plan to a user, to a group of users or sets the Global policy instance.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Grant-CsTenantDialPlan.md:16]:
```suggestion
Use the Grant-CsTenantDialPlan cmdlet to assign an existing tenant dial plan to a user, to a group of users, or to set the Global policy instance.
```

File: skype/skype-ps/skype/Grant-CsTenantDialPlan.md [hunk 17]
```diff
@@ -5,71 +5,102 @@ applicable: Microsoft Teams, Skype for Business Online
...
+```
+
+### Identity
+```
+Grant-CsTenantDialPlan [[-Identity] <string>] [[-PolicyName] <string>] [-PassThru] [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The Grant-CsTenantDialPlan cmdlet assigns an existing tenant dial plan to a user.
+The Grant-CsTenantDialPlan cmdlet assigns an existing tenant dial plan to a user, a group of users or sets the Global policy instance.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Grant-CsTenantDialPlan.md:36]:
```suggestion
The Grant-CsTenantDialPlan cmdlet assigns an existing tenant dial plan to a user, a group of users, or sets the Global policy instance.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @JensTrier, kindly commit the suggested changes. Many thanks.

File: skype/skype-ps/skype/Grant-CsDialoutPolicy.md [hunk 18]
```diff
@@ -156,18 +191,34 @@ Accept pipeline input: False
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
 ### CommonParameters
 This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (https://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Grant-CsDialoutPolicy.md:211]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: skype/skype-ps/skype/Grant-CsOnlineVoiceRoutingPolicy.md [hunk 19]
```diff
@@ -197,26 +184,40 @@ Accept pipeline input: False
...
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
 ### CommonParameters
 This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable.
 For more information, see about_CommonParameters (https://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Grant-CsOnlineVoiceRoutingPolicy.md:204]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: skype/skype-ps/skype/Grant-CsTeamsCallParkPolicy.md (hunk 13 above)
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Grant-CsTeamsCallParkPolicy.md:185]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: skype/skype-ps/skype/Grant-CsTeamsCallingPolicy.md [hunk 20]
```diff
@@ -142,6 +169,21 @@ Accept pipeline input: False
...
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
 ### CommonParameters
 This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable.
 For more information, see about_CommonParameters (https://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Grant-CsTeamsCallingPolicy.md:189]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable.
For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

**Comment by JensTrier** [comment]
> Copy review - @JensTrier, kindly commit the suggested changes. Many thanks.

Thanks! & done :-)

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tseward

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**